notebook/static-src/auth/js/loginwidget.js

```diff
@@ -1,12 +1,10 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
 
-define([
-    'base/js/utils',
-    'jquery',
-], function(utils, $){
     "use strict";
```

Comment:
Well this is cleaner.

I wonder if we should be declaring jquery global in the files in addition to the jshintrc.